src/main/java/core/basesyntax/fileservice/WriteToFile.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.fileservice;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriteToFile {
+    public void writeToFile(Map<String,Integer> fruitDao, String filePath) {
```

Comment:
same here. you can override toString() method in class with your storage and in this class just call it and write to file byte-to-byte